packaging/installer/README.md

```diff
@@ -165,3 +165,63 @@ the community helps fix any bugs that might have been introduced in previous rel
 -   Protect yourself from the rare instance when major bugs slip through our testing and negatively affect a Netdata
     installation
 -   Retain more control over the Netdata version you use
+
+## Installation notes and known issues
+
+We are tracking a few issues related to installation and packaging.
+
+### Failure to claim/connect to Netdata Cloud
+
+If you're running an older Linux distribution or one that has reached EOL, such as Ubuntu 14.04 LTS, Debian 8, or CentOS
+6/7, your Agent may not be able to connect to Cloud due to an old version of OpenSSL that is incompatible with Netdata
+Cloud.
+
+We recommend using the [static build installation](/packaging/installer/methods/kickstart-64.md) to avoid this issue.
+You can, however, use the workaround below.
+
+#### OpenSSL workaround
```

Comment:
Again Nit: Not a work-around.

Reply:
I'm fine with removing the work-around wording, it's good to make it clear.

Do note that users running on old machines they are most likely (hopefully) just using for testing Netdata won't care they are disabling security.